test/integration/integration.test

```diff
@@ -1,62 +1,30 @@
+<!-- 
+    This rostest file launches an integration test that tests basic
+    drive to waypoint functionality.
+ -->
 <launch>
-    <arg name="model" default="$(find mrover)/rover_description/rover.urdf"/>
-    <arg name="gui" default="false"/>
-    <param name="robot_description" command="xacro $(find mrover)/rover_description/rover.xacro"/>
-    <arg name="gazebo_config_path" default="$(find mrover)/config/gazebo"/>
+    <!-- launch rover core nodes -->
+    <include file="$(find mrover)/launch/rover_core.launch" />
 
-    <node if="$(arg gui)" name="joint_state_publisher" pkg="joint_state_publisher_gui"
-          type="joint_state_publisher_gui"/>
-    <node unless="$(arg gui)" name="joint_state_publisher" pkg="joint_state_publisher" type="joint_state_publisher"/>
-    <node name="robot_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher"/>
-
-    <include file="$(find gazebo_ros)/launch/empty_world.launch">
-        <env name="GAZEBO_MODEL_PATH" value="$(arg gazebo_config_path)/env_description"/>
-        <arg name="world_name" value="$(arg gazebo_config_path)/env_description/world.world"/>
-        <arg name="paused" value="false"/>
-        <arg name="use_sim_time" value="true"/>
-        <arg name="gui" value="false"/>
-        <arg name="headless" value="true"/>
-        <arg name="debug" value="false"/>
-        <arg name="verbose" value="false"/>
-        <arg name="gui_required" value="false"/>
+    <!-- launch the simulator with no GUI -->
+    <include file="$(find mrover)/launch/simulator.launch">
+        <arg name="gui" value="false" />
     </include>
 
-    <node name="spawn_model" pkg="gazebo_ros" type="spawn_model" output="screen" args="
-    -urdf -param robot_description
-    -model rover"/>
-
-    <node name="nav" pkg="mrover" type="navigation.py"/>
+    <!-- navigation -->
+    <node name="nav" pkg="mrover" type="navigation.py" />
```

Comment:
why is nav being launched here?

Reply:
bc rn this test doesnt use perception so I didnt launch the full auton.launch. in the future when we add that we can make this nicer

Reply:
lets do a flag for perception, I imagine we frequently will want to disable it for testing.

Reply:
done